src/pudl/transform/eia923.py

```diff
@@ -14,6 +14,9 @@
 ###############################################################################
 ###############################################################################
 
+TARGET_DROP_PLANT_IDS = [54587]
+MISSING_DATA_STRINGS = ["nan"]
+
```

Comment:
Does it make sense to have these be global variables? They're very specific to the cleaning of the generation table, and their names don't indicate what context they apply to. I think I'd be inclined to keep this information inside the `generation()` function so that they can't be picked up in any other context where they don't apply.

Reply:
Good point. I moved the variables into `generation()`.